.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - develop
+      - cspl-1985
```

Comment:
nit: hope we remove it before the merge?

Reply:
Will be adjusted to both the develop and main branch before merge.